### PR TITLE
docs: add basic usage to README

### PR DESCRIPTION
**Pull request by TinsFox**
# Update docs

## Description
As a new user, I prefer to see how it was done in the beginning rather than upgrading the documentation. This can feel strange to new users

### CODE REVIEW

**Comment by ayushmanchhabra** [comment]
I recently moved all documentation from README to JSDoc comments so that it can be autogenerated as a website. I don't really see an issue with not having install instructions on the README if a link to a docs site is given.

File: README.md [hunk 1]
```diff
@@ -19,35 +19,87 @@ Check out the [documentation](https://nwutils.io/nw-builder/) if you wish to giv
...
+1. To get nwjs cache
+
+    ```bash
+    nwbuild --mode=get
+    ```
+
+2. To run nwjs application
+
+    ```bash
+    nwbuild --mode=get
```
Comment (sysrage) [inline README.md:57]: Wrong flag for "run".

File: README.md [hunk 2]
```diff
@@ -19,35 +19,87 @@ Check out the [documentation](https://nwutils.io/nw-builder/) if you wish to giv
...
+        "build": "node scripts/build.js"
+      }
+    }
+    ```
+2. Create a build script
+    ```javascript
+    // scripts/build.js
+    const { nwbuild } = require("nw-builder");
+    await nwbuild({
+      srcDir: "./nwapp/**/* ./other/**/*.js",
```
Comment (sysrage) [inline README.md:81]: Is this correct? I don't use globs, but why is this a space-separated string rather than an array of globs?

**Review by sysrage — CHANGES_REQUESTED** [review]
I do agree basic usage instructions should be included in the README. The documentation can/should be used more as an API reference if/when basic usage isn't enough.

**Comment by ayushmanchhabra** [comment]
What do you mean by basic usage? I'm guess I'm too familiar with the options so everything to me seems like basic usage to me :P

**Comment by TinsFox** [comment]
> What do you mean by basic usage? I'm guess I'm too familiar with the options so everything to me seems like basic usage to me :P

It's how to use the library quickly to get him to do something that's expected. Just the most basic configuration is fine, probably the effect of a command like 'nwbuild --mode=build', if this command automatically fills in the other default parameters

**Comment by TinsFox** [comment]
> I recently moved all documentation from README to JSDoc comments so that it can be autogenerated as a website. I don't really see an issue with not having install instructions on the README if a link to a docs site is given.

I see that, but I recommend removing this line

https://github.com/nwutils/nw-builder/blob/3e367f383d439b850500bdffae641f83032014e8/.gitignore#L11

 because keeping the Markdown file allows us to add some extra instructions to it, rather than completely handing it over to the program

**Comment by ayushmanchhabra** [comment]
> > I recently moved all documentation from README to JSDoc comments so that it can be autogenerated as a website. I don't really see an issue with not having install instructions on the README if a link to a docs site is given.
> 
> I see that, but I recommend removing this line
> 
> https://github.com/nwutils/nw-builder/blob/3e367f383d439b850500bdffae641f83032014e8/.gitignore#L11
> 
> because keeping the Markdown file allows us to add some extra instructions to it, rather than completely handing it over to the program

I understand what you mean. This is how I initially managed the docs. I'm trying to automate as many things as I can so that the maintainer(s) after me will have it easier.

**Comment by ayushmanchhabra** [comment]
@TinsFox I've made a couple of commit suggestions

**Comment by TinsFox** [comment]
> @TinsFox I've made a couple of commit suggestions

I can't see your comment. Did you forget to submit it?

**Review by ayushmanchhabra — APPROVED** [review]
I'm fine with the changes here. I have to figure out how to setup docs nicely which I'll do in subsequent PRs.

**Pull request merged**